src/components/IndividualPageContent/ContentRowSpaceBetween.tsx

```diff
@@ -14,23 +14,15 @@ export default function ContentRowSpaceBetween({
   title,
   value,
   tooltip,
-  i,
 }: ContentRowSmallProps) {
   return (
     <Stack direction="row" justifyContent="space-between">
-      <Box sx={{fontSize: "0.875rem", color: grey[450]}}>
-        {title}
-        <Box
-          component="span"
-          sx={{
-            display: "inline",
-            whiteSpace: "nowrap",
-          }}
-        >
-          &nbsp;
-          <Box sx={{display: "inline-block"}}>{tooltip}</Box>
-        </Box>
-      </Box>
```

Comment:
updating the `ContentRowSpaceBetween` format to accommodate the layout. if we don't change it, tooltip and text are not inline vertically and spacing is weird.